setup.py

```diff
@@ -41,36 +41,32 @@ def read_requirements(file):
     return reqs.strip().split("\n")
 
 
-DISTNAME = "torchmultimodal"
-DESCRIPTION = "Multimodal modeling in PyTorch"
-LONG_DESCRIPTION = fetch_long_description()
-LONG_DESCRIPTION_CONTENT_TYPE = "text/markdown"
-AUTHOR = "PyTorch Multimodal"
-AUTHOR_EMAIL = "kartikayk@fb.com"
-# Need to exclude folders in test as well so as they don't create an extra package
-EXCLUDES = ("examples*", "test*")
-
-
 if __name__ == "__main__":
 
     setup(
-        name=DISTNAME,
+        name="torchmultimodal",
         include_package_data=True,
-        packages=find_packages(exclude=EXCLUDES),
+        packages=find_packages(
+            exclude=("examples*", "test*")
```

Comment:
should this become "tests" now?

Reply:
Yeah good catch, although `test*` can still capture `tests*` but let me change it anways.